src/main/scala/scorex/core/network/ModifiersStatus.scala

```diff
@@ -0,0 +1,33 @@
+package scorex.core.network
+
+object ModifiersStatus {
+
+  sealed trait ModifiersStatus
```

Comment:
I suggest moving this trait to the top level

Reply:
Now we have statuses like `ModifierStatus.Unknown` that describes it quite good. `scorex.core.network.Unknown` does not describe well, what are we talking about (we also have History status and so on.)